### ISSUE TITLE
GA Installer should be taken from bigfix internet site instead of internal build site

### ISSUE THREAD

**Issue by gearoidibm** [bigfix/boxes#9]
If the ServerInstaller tar ball is taken from software.bigfix.com instead of the build site it means this can be used without access to the build server

**Comment by gnawhleinad** [comment]
The repo is biased for internal usage.

The install script via [`server.sh`](https://github.com/bigfix/boxes/blob/master/bigfix/redhat/server.sh) depends on either `BIGFIX_VERSION` (which downloads from builds.sfolab.ibm.com) or a local installer (which you can download from software.bigfix.com).

That being said, we could make it easier to download installers from either source.

**Comment by gearoidibm** [comment]
I get you - if it was always using the bigfix.com site it wouldn't work for internal / non ga builds.
Maybe a way to easily switch between internal build site and bigfix.com as the source of the bigfix software ?

**Comment by gnawhleinad** [comment]
Yeah! [Thinking out loud](https://www.youtube.com/watch?v=f6Cswdm601A):

* Add an environment variable to configure where to download the installer from
* Update `BIGFIX_VERSION` logic to choose software.bigfix.com if it's a GA release
